Source/WebSocket/Websocketpp/websocketpp_websocket.cpp

```diff
@@ -606,7 +600,7 @@ struct wspp_websocket_impl : public hc_websocket_impl, public std::enable_shared
     };
 
     // Asio client has a long running "run" task that we need to provide a thread for
-    AsyncBlock* m_backgroundAsync;
+    std::thread m_websocketThread;
     async_queue_handle_t m_backgroundQueue;
```

Comment:
>m_backgroundQueue [](start = 25, length = 17)

this is unused, remove?

Reply:
It is used when shutdown_wspp_impl is called

---
In reply to: [217564415](https://github.com/Microsoft/libHttpClient/pull/279#discussion_r217564415) [](ancestors = 217564415)

Reply:
Good point, it is still needed. 

---
In reply to: [217564415](https://github.com/Microsoft/libHttpClient/pull/279#discussion_r217564415) [](ancestors = 217564415)